### PR TITLE
Right click / touch in mobile will make player move

### PR DESCRIPTION
**Pull request by aaron-lam**
Player can now control the hero by right clicking the ground / touching the ground using mobile devices.

### CODE REVIEW

**Comment by makkamchuen** [comment]
@chacobo said we may not need to check in meta file, lets discuss with him today. But the PR looks good to me

File: Assets/Game/Scripts/CameraController.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+
+public class CameraController : MonoBehaviour
```
Comment (makkamchuen) [inline Assets/Game/Scripts/CameraController.cs:3]: This setup always follow the character, right? I think it is good for now unless we want to have a minimap

**Review by makkamchuen — APPROVED** [review]

**Comment by aaron-lam** [comment]
> @chacobo said we may not need to check in meta file, lets discuss with him today. But the PR looks good to me

Sounds good! Will put this PR on hold then.

**Comment by chacobo** [comment]
So I found only some but not all metas could be ignored. Our .gitignore has the necessary items so we should be fine as it is regarding metas.

Here's a community-maintained .gitignore that we could use, which is a bit different than ours, but shouldn't produce real world difference.
https://github.com/github/gitignore/blob/master/Unity.gitignore.

**Review by chacobo — APPROVED** [review]

**Pull request merged**